status/index.test.ts

```diff
@@ -198,7 +198,8 @@ it('removes listeners', () => {
   })
 
   let calls = 0
-  let unbind = status(client, state => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  let unbind = status(client, (state , details) => {
```

Comment:
It is not a fix :)

Reply:
It was a compilation error before. Now it's a linter warning